### PR TITLE
REFACTOR-digitalmarketplace-api/app/main/views/users.py

### PR DESCRIPTION
**Pull request by benvand**
* Standardise imports
* Move helper function to live in utils file

### CODE REVIEW

**Comment by TheDoubleK** [comment]
Looks like you made Flake 8 sad :joy:

**Comment by benvand** [comment]
@TheDoubleK Whaaaaa?? It's flake8ing the git refs 🤔

**Comment by benvand** [comment]
@TheDoubleK https://github.com/alphagov/digitalmarketplace-api/pull/697

File: app/main/views/users.py [hunk 1]
```diff
@@ -1,16 +1,28 @@
...
+from ...utils import (
+    get_json_from_request, json_has_required_keys, json_has_matching_id, pagination_links, get_valid_page_or_1,
+    validate_and_return_updater_request
+)
+from ...validation import (
+    buyer_email_address_has_approved_domain, validate_user_auth_json_or_400, validate_user_json_or_400
+)
+
+
+def check_supplier_role(role, supplier_id):
```
Comment (TheDoubleK) [inline app/main/views/users.py]: If we're moving this round then maybe `supplier_utils.py` would be a better home for it?

**Comment by benvand** [comment]
Yeah good call, those utils files are a bit weird but there is another view helper in there. I think those files would make more sense in `app/main/views/utils/supplier_utils.py`. And then stuff like `app/service_utils.py::commit_and_archive_service` would live in `app/main/models/utils/service_utils.py`. Maybe that's overkill but feels like there's a logical split there. Not exactly the most urgent thing to do.

**Comment by benvand** [comment]
No probs @TheDoubleK :) this was just a bugbear

**Pull request merged**